packages/headless/src/controllers/result-list/headless-result-list.ts

```diff
@@ -64,6 +67,12 @@ export interface ResultListState extends SearchStatusState {
    * The unique identifier of the last executed search.
    */
   searchUid: string;
+  /**
+   * Whether more results are available, using the same parameters as the last successful query.
+   *
+   * This property is not compatible with the `Pager` controller.
```

Comment:
Is it worth it to add more complete documentation/explanation about infinite scrolling/fetch more result/pager not being compatible/etc not being compatible with one another ?

This seems to be a point of confusion.

Reply:
If I can jump in here, I think it's _definitely_ worth having more documentation on it, but I don't think this is the place to do it. We should have a page (or possibly multiple pages in a single section) on the documentation site about common use cases, and infinite scroll would be one of them. We do something similar for JSUI.

Reply:
So, I remove these comments? 😄

Reply:
Nah, I think you should leave what you have. It'll be particularly helpful until we get around to writing those documentation site sections 👍🏻